cmd/juno/juno.go

```diff
@@ -174,8 +174,9 @@ const (
 	dbCacheSizeUsage     = "Determines the amount of memory (in megabytes) allocated for caching data in the database."
 	dbMaxHandlesUsage    = "A soft limit on the number of open files that can be used by the DB"
 	gwAPIKeyUsage        = "API key for gateway endpoints to avoid throttling" //nolint: gosec
-	gwTimeoutUsage       = "Timeout for requests made to the gateway"          //nolint: gosec
-	callMaxStepsUsage    = "Maximum number of steps to be executed in starknet_call requests. " +
+	gwTimeoutsUsage      = "Timeouts for requests made to the gateway. Can be specified as a single value or comma-separated list" +
+		"or multiple flags. Values over 2m will be capped. Example: --gw-timeouts=5s,7s,10s"
```

Comment:
Modify this so that the following information is clear:
1. If only a number is set, after a fail retry the timeout will increase
2. If a list is set, the values of that list will use after a retry
3. To avoid the dynamic windows and setting a single value set it as `5s,` (trailing comma)